doc/source/installation.rst

```diff
@@ -6,14 +6,22 @@ Installation
 .. note::
 
     This document describes how to install Keg. Currently `keg` is
-    provided from `PyPi <https://pypi.org/project/kiwi_keg/>`__ and
-    further install methods for Linux distributions will follow
-    soon.
+    available from `PyPi <https://pypi.org/project/kiwi_keg/>`__ and
+    from the `openSUSE Build Service <https://build.opensuse.org/>`__
```

Comment:
Consider linking directly to the keg package in OBS.

Reply:
Agreed we should point directly to the project where we build keg. That said that link is provided below as such I think the link here is superfluous. If you want to highlight the build service then this should link to the GitHub repo fro OBS.

Reply:
I'm thinking maybe we should drop the installation instructions from the docs and put more of a generic "Keg packages are available from PyPi and for various openSUSE distributions as RPMs. Find details in [link to github repo]."

I'm guessing we want to submit keg to Factory eventually and perhaps also have it in PackageHub, which would leave that section a bit incomplete, so might be easier to put this info in the more easily updateable README. What do you think?

Reply:
I think that very much depends on where we expect the docs to show up. If the docs get included in SLE somewhere then the documentation should talk about PackageHub, which of course we still have to do. If the doc lands somewhere under "Best Practices" then then referencing OBS is IMHO good. As such I think we need some guidance from the doc team.

Reply:
It's currently listed on documentation.suse.com under SLES, which is perhaps not ideal. I'll reach out to the doc team. For the meantime, I've included the direct link to the OBS package.